### ISSUE TITLE
ResourcefulBees no longer working

### ISSUE THREAD

**Issue by crazycatlord117** [AllTheMods/ATM-6#1407]
The ability for all but the vanilla honeycomb to be processed through the centrifuge is gone, as in JEI no longer has the 'recipe' and while you can put the resourceful bees honeycomb in the centrifuge it just does nothing.

**Comment by Paper-28** [comment]
Hi read #1384 issue

**Comment by whatthedrunk** [comment]
Yeah a duplicate issue. It looks fixed on their end just waiting for the mod to update on Curseforge now.
